meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -163,6 +163,22 @@ MBL_WKS_BOOTLOADER2_IS_BANKED = "1"
 MBL_WKS_BOOTLOADER2_NO_FS = "1"
 MBL_PARTITION_NAMES += "WKS_BOOTLOADER2"
 
+# ------------------------------------------------------------------------------
+# BANK_AND_UPDATE_STATE (non-file system; 1 bank)
```

Comment:
Should we follow the standard and have `single` instead of `1`?

Reply:
Yes, we should: https://github.com/ARMmbed/meta-mbl/pull/671/commits/b1585c233087454720525b2276eec828a7780681.